src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/Helpers.kt

```diff
@@ -1,22 +1,33 @@
 package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service
 
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.data.MigrationContext
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.data.SynchronisationContext
 import java.time.LocalDate
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
 
 fun LocalDateTime?.asStringOrBlank(): String = this?.format(DateTimeFormatter.ISO_DATE_TIME) ?: ""
 fun LocalDate?.asStringOrBlank(): String = this?.format(DateTimeFormatter.ISO_DATE) ?: ""
 
-class MigrationMessage<M, T>(
```

Comment:
should have called this Message but it clashed with too many other classes called Message to panicked and called it this instead